<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QMainWindow" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>216</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>182</width>
    <height>216</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>182</width>
    <height>216</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Point</string>
  </property>
  <property name="windowIcon">
   <iconset resource="point.qrc">
    <normaloff>:/icons/point</normaloff>:/icons/point</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget { background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.509852, y2:0, stop:0.173267 rgba(234, 234, 234, 255), stop:1 rgba(255, 255, 255, 255)); }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="pointLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="point.qrc">:/icons/point</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="loginLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="errorLabel">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string notr="true">Error text</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailText">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="placeholderText">
           <string>E-mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordText">
          <property name="inputMask">
           <string/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="rememberBox">
          <property name="text">
           <string>Remember login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Log in</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>emailText</tabstop>
  <tabstop>passwordText</tabstop>
 </tabstops>
 <resources>
  <include location="point.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>passwordText</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emailText</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
